client/components/AccountCreation/AccountCreation.jsx

```diff
@@ -0,0 +1,127 @@
+import React, {Component} from 'react';
+
+//Default values for form data
+const initialFormData = Object.freeze({
+    username: "",
+    password: "",
+    confirmPassword: ""
+});
+
+function AccountCreation(props) {
+    //formData is an object that holds username, password, confirmPassword
+    const [formData, updateFormData] = React.useState(initialFormData);
+
+    //Whenever username or confirmPassword input boxes change, this saves the new data to formData
+    const handleChange = (e) => {
+        updateFormData({
+            ...formData,
+            [e.target.name]: e.target.value.trim()
+        });
+    }
+    //Whenever the submit button is clicked, this checks to make sure the passwords match and calls another func
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        if(formData.password !== formData.confirmPassword) {
+            alert("YOUR PASSWORDS DO NOT MATCH");
+        }
+        else if(checkPasswordRequirements(formData.password) && checkUsernameRequirements(formData.username)) {
+            //TODO send the formData to the backend
+            console.log("Hey this code works");
+        }
+    }
+
+    return (
+        <div>
+            <form onSubmit={handleSubmit}>
+                <label>
+                    Username:
+                    <br/>
+                    <input onChange={handleChange} data-testid="username" name="username"/>
+                </label>
+                <label>
+                    <br/>
+                    Password:
+                    <br/>
+                    <input onChange={handleChange} data-testid="password" type="password" name="password"/>
+                </label>
+                <label>
+                    <br/>
+                    Confirm Password:
+                    <br/>
+                    <input onChange={handleChange} data-testid="confirmPassword" type="password" name="confirmPassword"/>
+                </label>
+                <br/>
+            </form>
+            <button onClick={handleSubmit} data-testid="submitButton">Submit</button>
```

Comment:
see LandingPage.jsx for example of React component Form by Ella. It'd keep things a little more consistent :D

Reply:
done